authorization/v1/types.go

```diff
@@ -537,6 +537,7 @@ type ClusterRoleList struct {
 // +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/470
 // +openshift:file-pattern=cvoRunLevel=0000_03,operatorName=config-operator,operatorOrdering=01
 // +openshift:compatibility-gen:level=1
+// +kubebuilder:metadata:annotations=release.openshift.io/bootstrap-required=true
 type RoleBindingRestriction struct {
```

Comment:
oh neat, I had forgotten about this one.  I wonder how many people use this feature.